src/multitfa/core/reaction.py

```diff
@@ -261,6 +262,60 @@ def calculate_transport_charge(self):
             {key: sum(val) for key, val in n_proton.items()},
         )
 
+    def update_variable_bounds(self):
+        """Updating the method from cobra to update the flux bounds in cplex_interface. Currently not doing it for gurobi_interface as I don't have a licence to test this"""
```

Comment:
I have a Gurobi license, could give it a try and make a PR to this branch or in a follow-up after this is merged. Or would you rather stop support for gurobi? My use case for gurobi so far has been that the constraint relaxation is more flexible, but I have managed to get CPLEX to do what I want. Not sure if the ["Debugging" part of the docs](https://multitfa.readthedocs.io/en/latest/additional_information.html) would still work, although I don't _think_ that gurobi's `computeIIS` function takes the quadratic constraint into account anyways.